packages/react-next/src/components/Toggle/useToggle.ts

```diff
@@ -128,6 +130,7 @@ export const useToggle = (
       'aria-labelledby': labelledById,
       className: classNames.pill,
       'data-is-focusable': true,
+      'data-ktp-target': true,
```

Comment:
hmm not clear why this is here.

Reply:
this is needed for specifying target element when it's not the root element